src/amo/components/LandingPage.js

```diff
@@ -55,13 +55,16 @@ export class LandingPageBase extends React.Component {
         popularHeader: i18n.gettext('Most popular extensions'),
         popularFooterLink: {
           pathname: '/search/',
-          query: { sort: SEARCH_SORT_POPULAR, type: ADDON_TYPE_EXTENSION },
+          query: { addonType: ADDON_TYPE_EXTENSION, sort: SEARCH_SORT_POPULAR },
```

Comment:
Aha so this was being set but was transformed elsewhere? Would be good to have a regression test to cover this.